dev/build.image/profiles/jakartaee10/features.xml

```diff
@@ -1,5 +1,7 @@
 <feature>jakartaee-10.0</feature>
 <feature>jakartaeeClient-10.0</feature>
+<feature>jdbc-4.2</feature>
+<feature>jdbc-4.3</feature>
```

Comment:
Both versions?

Reply:
That is the point.  Today the umbrella features have 4.2 tolerates 4.3.  If I just have 4.3, it will only include 4.3.  The spec talks about the 4.2 API so it is required.  What Fred asked for is to include both 4.2 and 4.3 in the packages.

Reply:
this got me too